commands/index.js

```diff
@@ -1,12 +1,11 @@
-module.exports = function index(pkg) {
+module.exports = function index (pkg) {
   return {
     topic: pkg.topic,
     description: pkg.description,
-    help: pkg.description,
```

Comment:
an example of it printing the version then, we're supposed to have output examples for every command. I realize that isn't the most useful thing in the world, but it clearly demonstrates what a command is for.

this is the (soon-to-be enforced) rule for core plugins anyways, but we should do it on any plugin we can